.gitignore

```diff
@@ -227,3 +227,6 @@ vda/vdad
 build/
 build-*/
 
+# WASM
+*.wasm
+a.out.js
```

Comment:
It would be nice to have output file be `boinc.js` instead of `a.out.js`. Is it possible to change it?

Reply:
The binary is boinc and boinc.wasm. this is only intermediate file. 
boinc is a JavaScript file.